### ISSUE TITLE
Testnet should support https

### ISSUE THREAD

**Issue by njordhov** [blockstack/stacks-transactions-js#114]
**Describe the bug**

Testnet uses `http` protocol rather than `https`. This may cause issues for dapps served with an `https` protocol.

**Steps To Reproduce**

https://github.com/blockstack/stacks-transactions-js/blob/e86e2b11be27612532aa5ca131d036aa89190c12/src/network.ts#L51

**Expected behavior**

Testnet has an url with `https` protocol.

**Comment by hstove** [comment]
Yeah - in the authenticator, we are overriding this URL to use the HTTPS sidecar URL. https://github.com/blockstack/ux/blob/fix/570_postconditions/packages/keychain/src/wallet/signer.ts#L68:L70
